app/services/idv/steps/send_link_step.rb

```diff
@@ -97,6 +97,7 @@ def form_submit
       def formatted_destination_phone
         raw_phone = permit(:phone)[:phone]
         PhoneFormatter.format(raw_phone, country_code: 'US')
+        idv_session[:phone] = raw_phone
```

Comment:
Are there other consumers of the `formatted_destination_phone` method? Before, it was returning the `PhoneFormatter` version of the phone number, but with the session assignment I don't think it would be returning that anymore

Reply:
you are right, this is in the wrong place!